drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -6508,6 +6508,9 @@ void RasterizerStorageGLES2::initialize() {
 	GLOBAL_DEF_RST("rendering/quality/lightmapping/use_bicubic_sampling.mobile", false);
 	config.use_lightmap_filter_bicubic = GLOBAL_GET("rendering/quality/lightmapping/use_bicubic_sampling");
 
+	GLOBAL_DEF_RST("rendering/quality/shading/use_physical_light_attenuation", false);
```

Comment:
Small nitpick, but we have been trying to define project settings in the VisualServer so that they are all in one place and are visible when using either rendering backend.